wasm2c/wasm-rt.h

```diff
@@ -403,6 +432,26 @@ uint64_t wasm_rt_grow_memory(wasm_rt_memory_t*, uint64_t pages);
  */
 void wasm_rt_free_memory(wasm_rt_memory_t*);
 
+#ifdef WASM_RT_C11_AVAILABLE
+/**
+ * Shared memory version of wasm_rt_allocate_memory
+ */
+void wasm_rt_allocate_shared_memory(wasm_rt_shared_memory_t*,
```

Comment:
Document the new runtime API functions in wasm2c/README.md?

Reply:
Done